src/Arcus.Observability.Telemetry.Core/ContextProperties.cs

```diff
@@ -0,0 +1,46 @@
+﻿#pragma warning disable 1591
+namespace Arcus.Observability.Telemetry.Core
+{
+    public class ContextProperties
```

Comment:
All static classes? Maybe no need to use XML docs, idk, I'll leave it up to you.

Reply:
It's for a more fluent approach of using them but open for suggestions

Reply:
Can't you create static classes inside static classes?